src/handlers/comment/handlers/wallet.ts

```diff
@@ -34,7 +34,7 @@ export const registerWallet = async (body: string) => {
 
   if (!address && !ensName) {
     logger.info("Skipping to register a wallet address because both address/ens doesn't exist");
-    return;
+    return `Wallet address or ens do not exist, Please try again \n usage: /wallet 0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266`;
```

Comment:
```suggestion
    return `Wallet address or ens do not exist, Please try again \n usage: /wallet 0x0000000000000000000000000000000000000000`;
```

Can you update all examples to this `0x0000000000000000000000000000000000000000` address so that we can catch this in the assign command easily?